demo/src/test/java/com/project/demo/repository/UserRepositoryTests.java

```diff
@@ -125,60 +125,35 @@ public void infoSetup() {
     }
 
     @Test
-    public void UserRepository_findByUsername_ReturnsUser() {
-        Optional<User> userFoundOptional = userRepository.findByName("John");
+    public void findUserToResetPassword() {
+        Optional<User> userFoundOptional = userRepository.findByEmail("test@gmail.com");
         Assertions.assertThat(userFoundOptional).isPresent();
-        User userFound = userFoundOptional.get();
-        Assertions.assertThat(userFound).isNotNull();
-        Assertions.assertThat(userFound.getName()).isEqualTo("John");
-    }
-
-    @Test
-    public void testLoginSuccess() {
```

Comment:
Watch out for deleting existing tests.

Reply:
Iván added again the missing tests. Resolving comment.